components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -386,113 +381,3 @@ export function PrebuildStatus(props: { prebuild: PrebuildWithStatus }) {
         </div>
     );
 }
-
-// Deprecated. Use PrebuildStatus instead.
-export function PrebuildInstanceStatus(props: { prebuildInstance?: WorkspaceInstance; prebuild?: PrebuildWithStatus }) {
-    let status = <></>;
-    let details = <></>;
-    switch (props.prebuildInstance?.status.phase) {
-        case undefined: // Fall through
-        case "preparing": // Fall through
-        case "pending": // Fall through
-        case "creating": // Fall through
-        case "unknown":
-            status = (
-                <div className="flex space-x-1 items-center text-yellow-600">
-                    <img alt="" className="h-4 w-4" src={StatusPaused} />
-                    <span>PENDING</span>
-                </div>
-            );
-            details = (
-                <div className="flex space-x-1 items-center text-gray-400">
-                    <img alt="" className="h-4 w-4 animate-spin" src={Spinner} />
-                    <span>Preparing prebuild ...</span>
-                </div>
-            );
-            break;
-        case "initializing": // Fall  through
-        case "running": // Fall through
-        case "interrupted": // Fall through
-        case "stopping":
-            status = (
-                <div className="flex space-x-1 items-center text-blue-600">
-                    <img alt="" className="h-4 w-4" src={StatusRunning} />
-                    <span>RUNNING</span>
-                </div>
-            );
-            details = (
-                <div className="flex space-x-1 items-center text-gray-400">
-                    <img alt="" className="h-4 w-4 animate-spin" src={Spinner} />
-                    <span>Prebuild in progress ...</span>
```

Comment:
**suggestion**: Probably not the best line to comment this and sounds like out of the scope of this PR but what do you think of changing the prebuild phase of the workspace start page by reusing the prebuild elements? Cc @jankeromnes @AlexTugarev because of relevant past discusions. Probably also best as a follow up issue.

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-04-05 at 2 00 24 PM" src="https://user-images.githubusercontent.com/120486/161750940-28214fab-25d4-4306-bd85-2a1fe347b6a7.png"> | <img width="1440" alt="Screenshot 2022-04-05 at 2 09 51 PM" src="https://user-images.githubusercontent.com/120486/161750944-0304302b-bbf4-4479-91a0-c0857de35921.png"> |

Reply:
This makes sense, would definitely help with increased perceived reliability as the UI would be consistent. I've cut an issue for this. https://github.com/gitpod-io/gitpod/issues/9132

Reply:
Je vous remercie, @easyCZ! ✨